mtiapko/lab1-BasicIO/README.md

```diff
@@ -0,0 +1,29 @@
+# LAB #1: Basic of peripheral I/O ports
+
+## Content
+* [About](#about)
+* [Build](#build)
+
+## About
+
+Programs written in this work by pressing the button `BTN_1` lights one of the four LEDs on the board. LEDs lights occurs cyclically.
+
+| # |LED off|LED on |
+|:-:|:-----:|:-----:|
+|1. |   -   |`LED_1`|
+|2. |`LED_1`|`LED_2`|
+|3. |`LED_2`|`LED_3`|
+|4. |`LED_3`|`LED_4`|
+|5. |`LED_4`|`LED_1`|
+|6. |`LED_1`|  ...  |
+
+The project was written and tested on a microcontroller `PIC32MZ2048EFG100`.
+
+## Build
+
+![build img](./build.jpg "Building process")
```

Comment:
great!
There's a trick on using images on Github without adding them to repository [link](http://solutionoptimist.com/2013/12/28/awesome-github-tricks/)